codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -222,17 +216,11 @@ public int onDataRead(final ChannelHandlerContext ctx, int streamId, ByteBuf dat
             int unconsumedBytes = unconsumedBytes(stream);
             Http2LocalFlowController flowController = flowController();
```

Comment:
Now that the flow controller is being used higher up.  Move this variable up so it can be used with the new code.